applications/LS-Dyna-stampede3/app.json

```diff
@@ -0,0 +1,124 @@
+{
+    "id": "LS-Dyna-stampede3",
+    "version": "2024R1",
+    "description": "LS-DYNA is an advanced general-purpose multiphysics simulation software package.",
+    "owner": "${apiUserId}",
+    "enabled": true,
+    "runtime": "ZIP",
+    "runtimeVersion": null,
+    "runtimeOptions": null,
+    "containerImage": "tapis://cloud.data//corral/tacc/aci/CEP/applications/v3/ls-dyna-stampede3/ls-dyna-stampede3.zip",
+    "jobType": "BATCH",
+    "maxJobs": -1,
+    "maxJobsPerUser": -1,
+    "strictFileInputs": true,
+    "jobAttributes": {
+        "description": "",
+        "dynamicExecSystem": false,
+        "execSystemConstraints": null,
+        "execSystemId": "stampede3",
+        "execSystemExecDir": "${JobWorkingDir}",
+        "execSystemInputDir": "${JobWorkingDir}",
+        "execSystemOutputDir": "${JobWorkingDir}",
+        "execSystemLogicalQueue": "skx-dev",
+        "archiveSystemId": "cloud.data",
+        "archiveSystemDir": "HOST_EVAL($HOME)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}",
+        "archiveOnAppError": true,
+        "isMpi": false,
+        "mpiCmd": null,
+        "cmdPrefix": null,
+        "parameterSet": {
+            "appArgs": [
+                {
+                    "name": "Input file",
+                    "description": "Please enter the input file in .txt format. Add one .k case file per line if there are multiple case files.",
+                    "arg": "myfilename.txt",
+                    "inputMode": "REQUIRED",
+                    "notes": {
+                        "validator": {
+                            "regex": "\\.txt$",
+                            "message": "The input file must be a .txt file."
+                        }
+                    }
+                }
+            ],
+            "containerArgs": [],
+            "schedulerOptions": [
+                {
+                    "name": "TACC Scheduler Profile",
+                    "description": "Scheduler profile for HPC clusters at TACC",
+                    "inputMode": "FIXED",
+                    "arg": "--tapis-profile ls-dyna-2023R2",
```

Comment:
The profile version differs from the app version. Is that intentional?

Reply:
Updated the profile name for both frontera and stampede3.